src/components/Form/Form.tsx

```diff
@@ -0,0 +1,161 @@
+/* eslint-disable */
+import React, { FC, useState } from "react";
+import { Formik, Field } from "formik";
+import {
+  Button,
+  LinearProgress,
+  MenuItem,
+  Snackbar,
+  FormControl,
+  InputLabel,
+} from "@material-ui/core";
+import { TextField, Select } from "formik-material-ui";
+import * as yup from "yup";
+import FormContainer from "components/FormContainer/FormContainer";
+import {
+  createStyles,
+  makeStyles,
+  useTheme,
+  Theme,
+} from "@material-ui/core/styles";
+
+export interface FormData {
+  email?: string;
+  password?: string;
+  select?: string;
```

Comment:
why does this global form component have `email`, `password` and `select`, then how will we reuse this across the project? this should be a generic component